Header header

geometry_msgs/Pose vehicle
geometry_msgs/Pose center
geometry_msgs/Vector3 size